docs/concepts.md

```diff
@@ -197,25 +197,20 @@ compares both confidentiality and integrity:
 - the **integrity** check ensures that the value sent is trusted by the
   destination
 
-More concretely, if `L_a = (C_a, I_a)` (where `C_a` and `I_a` are the
-confidentiality and integrity components respectively), and `L_b = (C_b, I_b)`
-then `L_a ⊑ L_b` iff `C_a ⊆ C_b` and `I_a ⊇ I_b`. Notably, the integrity part of
-the check is "flipped" by convention (see "Integrity Considerations for Secure
-Computer Systems" below). More fundamental than adhering to convention, by
-flipping the integrity order in this way, we can retain the intuitive meaning
-behind integrity tags as representing trust by a principal (trust by a user,
-trust on a subject, etc.).
+Intuitively, data can only flow from a source to a destination if:
 
-Intuitively, data can only flow from `a` to `b` if:
+- the destination is **at least as secret** as the source
+- the source is **at least as trusted** as the destination
```

Comment:
Moves this below the sections defining the confidentiality and integrity checks, imho I consider it part of their definitions.